lib/internal/bootstrap_node.js

```diff
@@ -206,7 +206,9 @@
     const util = NativeModule.require('util');
 
     // Deprecate GLOBAL and root
-    ['GLOBAL', 'root'].forEach(function(name) {
+    const deprecation = ['GLOBAL', 'root'];
+    for (var i = 0; i < deprecation.length; i++) {
```

Comment:
You can avoid repeated property access like this for a tiny perf boost:

`for (var i = 0, len = deprecation.length; i < len; i++)`